### ISSUE TITLE
Please add icon layout view masonry. Because i think this support icon mode view better

### ISSUE THREAD

**Issue by tienpd** [FortAwesome/Font-Awesome#7865]
Please add icon layout view masonry. Because i think this support icon mode view is better.

![layout-display-view-masonry-128](https://cloud.githubusercontent.com/assets/8694172/11057769/5458bf1e-87c0-11e5-957d-61f8c27876e3.png)

**Comment by MrGrigri** [comment]
+1

**Comment by ajmaske** [comment]
+100000   ;)

**Comment by dnajar2** [comment]
+1

**Comment by Gaetan-dc** [comment]
+1

**Comment by Latchy** [comment]
+1 Would be really useful and is super simple.

**Comment by bartclaeys** [comment]
Definitely need this! But I would call the above: `grid-cols-masonry`.

This implies a bunch of other icons:
`grid-cols`
`grid-rows`
`grid-rows-masonry`
`grid-cols-masonry`

**Comment by brbedk** [comment]
Yes!! now in 2020 - please, please please add this - as well as the grid-layout-horisontal #15239

**Comment by damianfok** [comment]
+1
